<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sound_settings_widget</class>
 <widget class="QWidget" name="sound_settings_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="driverGroupBox">
     <property name="title">
      <string>Driver</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="autoRadioButton">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="openalRadioButton">
        <property name="text">
         <string>OpenAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fmodRadioButton">
        <property name="text">
         <string>FMod</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="xaudio2RadioButton">
        <property name="text">
         <string>XAudio2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="directsoundRadioButton">
        <property name="text">
         <string>DirectSound</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="parametersGroupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="soundCheckBox">
        <property name="text">
         <string>Enable sound</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="eaxCheckBox">
        <property name="text">
         <string>Enable EAX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="softwareCheckBox">
        <property name="text">
         <string>Software sound buffer (may increase FPS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="soundTracksLabel">
        <property name="text">
         <string>Sound tracks</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="soundTracksHorizontalLayout">
        <item>
         <widget class="QSlider" name="tracksSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>4</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="tracksLabel">
          <property name="text">
           <string>8 tracks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
